lib/Dist/Zilla/App/Command/build.pm

```diff
@@ -16,6 +16,38 @@ does all the things required to build your distribution.  By default, it will
 also archive your distribution and leave you with a complete, ready-to-release
 distribution tarball.
 
+To go a bit further in depth, the C<build> command will do two things:
+
+=over
+
+=item
+
+Generate a directory containing your module, C<Foo-0.100>. This directory is
+complete. You could create a gzipped tarball from this directory and upload it
+directly to C<PAUSE> if you so desired. You could C<cd> into this directory and
+test your module so you're certain everything is as it should be before
+uploading to C<PAUSE>.
```

Comment:
Ah, I wasn't trying to lead people down that path at all, but rereading it I can certainly see how it could be taken that way.  I personally test this way on multiple perls with ```perlbrew exec ...``` but I don't upload to PAUSE from the directory. I upload from the gzipped tarball that's created at the same time.
My goal was to get across the point that the freshly generated directory is _exactly_ what you would get from the tarball or what would be uploaded if you did a ```dzil release```.  I'll mull over how to word that a bit more clearly.